Intersect.Server/Entities/BankInterface.cs

```diff
@@ -196,53 +196,26 @@ public List<int> FindItemSlots(Guid itemId, int quantity = 1)
         /// <returns>Returns whether or not the bank instance can store this item.</returns>
         public bool CanStoreItem(Item item)
         {
-            if (item.Descriptor != null)
+            // We don't want to store null items !
+            if (item.Descriptor == null)
             {
-                // Is the item stackable?
-                if (item.Descriptor.IsStackable)
-                {
-                    // Does the bank have this item already?
-                    var slot = FindItemSlot(item.ItemId);
-                    if (slot >= 0)
-                    {
-                        var existingSlot = mBank[slot];
+                return false;
+            }
 
-                        // Can we blindly add more to this stack?
-                        var untilFull = item.Descriptor.MaxBankStack - existingSlot.Quantity;
-                        if (untilFull >= item.Quantity)
-                        {
-                            return true;
-                        }
+            // If this item isn't stackable we just need to find an open slot.
+            if (!item.Descriptor.Stackable)
+            {
+                return FindOpenSlot() != -1;
+            }
 
-                        // Check to see if we have the inventory spaces required to hand these items out AFTER filling up the existing slot!
-                        var toGive = item.Quantity - untilFull;
-                        if (Math.Ceiling((double)toGive / item.Descriptor.MaxBankStack) <= FindOpenSlots().Count)
-                        {
-                            return true;
-                        }
-                    }
-                    // User doesn't have this item yet.
-                    else
-                    {
-                        // Does the user have enough free space for these stacks?
-                        if (Math.Ceiling((double)item.Quantity / item.Descriptor.MaxBankStack) <= FindOpenSlots().Count)
-                        {
-                            return true;
-                        }
-                    }
-                }
-                // Not a stacking item, so can we contain the amount we want to give them?
-                else
-                {
-                    if (FindOpenSlots().Count >= item.Quantity)
```

Comment:
This case appears to not be covered in the new code?

Reply:
This should be covering it when it comes to non stackable items:

https://github.com/AscensionGameDev/Intersect-Engine/blob/0765628603c5e32890fc7757e6dc79f2394fa100/Intersect.Server/Entities/BankInterface.cs#L205-L209

Reply:
Yeah but the code specifically said `>= item.Quantity` not `1` -- it makes me think there's a case where something can be not stackable with a quantity > 1...

Reply:
that would be weird (?) but just to make sure, i can add this check back, although i would have to see how it behaves with this... which tbh feels weird, right now we are just looking away for an available slot whenever it comes to stackables, since they aren't supposed to ... stack together in a single a slot (?)